modules/subscriptions.php

```diff
@@ -630,11 +630,14 @@ function comment_subscribe_init() {
 
 		// Check if Mark Jaquith's Subscribe to Comments plugin is active - if so, suppress Jetpack checkbox
 
-		$str = '';
+		$unique_id = esc_attr( uniqid( 'jetpack-subscription-form-' ) );
+
+		// Add heading and wrap in list for assistive technology
+		$str = '<div><h4 id="'. esc_attr( $unique_id ).'" class="comment-subscription-form-title">Comment subscription preferences</h4><ul class="comment-subscription-form-list" aria-labelledby="'. esc_attr( $unique_id ).'">';
```

Comment:
The "Comment subscription preferences" header needs to be translated, please use the gettext function for this:
```
__( 'Comment subscription preferences', 'jetpack' )
```